app/models/like.rb

```diff
@@ -0,0 +1,16 @@
+class Like < ApplicationRecord
+  belongs_to :author, class_name: 'User', foreign_key: :author_id
+  belongs_to :post
+
+  after_save :udate_counter_likes
+
+  def udate_counter_likes
+    post.update(likes_counter: post.likes.count)
+  end
```

Comment:
- [ ] [OPTIONAL] You're using a duplicated method, kindly make sure to remove this code from the Like model

```suggestion

```

Reply:
Hi, @Ismailco Thanks for the update. it was by mistake. I will do it and thanks for the approval.